docs/user/backup-and-restore.md

```diff
@@ -0,0 +1,242 @@
+# FlightCtl Backup and Restore Operations
+
+This document provides step-by-step instructions for backing up and restoring FlightCtl data, including running the `flightctl-restore` command to restore FlightCtl data from backups.
+
+## Overview
+
+The `flightctl-restore` command is used to prepare devices and data after restoring FlightCtl from a backup. This process requires temporarily stopping FlightCtl services, accessing the database and KV store directly, and then restarting services.
+
+⚠️ **Important**: This operation should only be performed during maintenance windows as it requires stopping all FlightCtl services.
+
+## Prerequisites
+
+- Access to the Kubernetes cluster where FlightCtl is deployed
+- `kubectl` configured with appropriate permissions
+- FlightCtl CLI tools available
```

Comment:
`redis-cli` is also a requisite, or it should be marked as optional later.

Reply:
added as an optional req

Reply:
redis cli is now using the rabbit suggested way to verify